errors/runconfig.go

```diff
@@ -0,0 +1,204 @@
+package errors
+
+// This file contains all of the errors that can be generated from the
+// docker/daemon component.
+
+import (
+	"net/http"
+
+	"github.com/docker/distribution/registry/api/errcode"
+)
+
+var (
+	// ErrorCodeInvalidMacAddress is generated when invalid Mac Address is provided.
+	ErrorCodeInvalidMacAddress = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "INVALIDMACADDRESS",
```

Comment:
@tiborvass do you mean something like INVALID-MAC-ADDRESS to be more readable? I like to have string constants in one single case to be easy to remember(no Camel-case)